.github/workflows/publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish to PyPI
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publish:
+    name: Publish to PyPI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v3
+        with:
+          python-version-file: .github/workflows/.python-version
+
+      - name: Install build dependencies
+        run: pip install build
+
+      - name: Build package
+        run: python -m build
+
+      - name: Publish to Test PyPI
```

Comment:
We need some additional parameters to correctly publish to Test PyPI. 

See https://github.com/pypa/gh-action-pypi-publish#advanced-release-management

And for example: https://github.com/compilerla/conventional-pre-commit/blob/main/.github/workflows/publish.yml#L26

Reply:
Ah, thanks for catching that. Added the missing parameters (and fixed the `user` parameter) in 3fd33a1c924d12cc936205b1b042cb6371492cf5